tetrateci/patches/eks/eks_1.15.patch

```diff
@@ -1,8 +1,21 @@
+From a2e8b635a71ac66273edd6388ba883a0214b9cf8 Mon Sep 17 00:00:00 2001
```

Comment:
for a long term, we can try to push this into upstream if this's a common solution to run test on eks.

Reply:
sure.